src/status_im/ethereum/json_rpc.cljs

```diff
@@ -72,6 +72,7 @@
    "wakuext_sendContactUpdate" {}
    "wakuext_sendContactUpdates" {}
    "wakuext_chats" {}
+   "wakuext_someChats" {}
```

Comment:
Why `someChats`? Is that like a set of random chats? :)

Reply:
any better suggestions? it's not all but some `several, a few, some, rather, something` :)

Reply:
`latestChats`, `latestNChats`, that's for getting chats with more recent updates, correct?

Reply:
`latestActiveChats`, idk. `someChats` sounds too random imo

Reply:
latestActiveChats looks good, thanks